### PR TITLE
chown: restore SUID and SGID bits

### PR DESCRIPTION
**Pull request by giuseppe**
be sure the SUID and SGID bits are not lost when we do
a chown.

Closes: https://github.com/containers/libpod/issues/1526

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by nalind** [comment]
LGTM

File: drivers/chown_unix.go [hunk 1]
```diff
@@ -45,10 +45,20 @@ func platformLChown(path string, info os.FileInfo, toHost, toContainer *idtools.
 			uid, gid = mappedPair.UID, mappedPair.GID
 		}
 		if uid != int(st.Uid) || gid != int(st.Gid) {
+			stat, err := os.Lstat(path)
```
Comment (kolyshkin) [inline drivers/chown_unix.go]: Maybe I'm missing something here, but I don't understand the reason for calling `os.Lstat()` here -- this function is already called with the `info` parameter, which is result of `os.Lstat(path)`.

Comment (giuseppe) [reply drivers/chown_unix.go]: good catch, I agree we can avoid `Lstat` here

**Pull request merged**